### PR TITLE
Add support for Python 3.9.

### PR DESCRIPTION
**Pull request by icemac**
It currently breaks 30 tests on Python 3.9 with something like:

```python

Traceback (most recent call last):
  File ".../3.9/lib/python3.9/unittest/case.py", line 59, in testPartExecutor
    yield
  File ".../3.9/lib/python3.9/unittest/case.py", line 593, in run
    self._callTestMethod(testMethod)
  File ".../3.9/lib/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File ".../Products.GenericSetup/Products/GenericSetup/tests/test_components.py", line 350, in test_blacklist_get
    body = adapted.body
  File ".../Products.GenericSetup/Products/GenericSetup/utils.py", line 525, in _exportBody
    xml = self._doc.toprettyxml(' ', encoding=self._encoding)
  File ".../Products.GenericSetup/Products/GenericSetup/utils.py", line 437, in toprettyxml
    return Document.toprettyxml(self, indent, newl, encoding)
  File ".../3.9/lib/python3.9/xml/dom/minidom.py", line 60, in toprettyxml
    self.writexml(writer, "", indent, newl, encoding, standalone)
TypeError: writexml() takes from 2 to 6 positional arguments but 7 were given
```

I did not have the time to investigate further.

### CODE REVIEW

**Comment by jamadden** [comment]
The `standalone` argument was added in [this commit in Python 3.9](https://github.com/python/cpython/commit/dc04a0571e362cd3de040771d7705cb107ae26fc). It *is* [newly-documented](https://docs.python.org/3/library/xml.dom.minidom.html), but that isn't specifically called out as a change.

**Comment by jamadden** [comment]
We could possibly just add that keyword argument in `_Element.writexml` and ignore it?

**Comment by icemac** [comment]
Coveralls decreased because we now only report Python 3 coverage.

File: src/Products/GenericSetup/PluginIndexes/exportimport.py [hunk 1]
```diff
@@ -13,15 +13,14 @@
...
-from zope.component import queryMultiAdapter
-
 from Products.PluginIndexes.interfaces import IDateIndex
 from Products.PluginIndexes.interfaces import IDateRangeIndex
 from Products.PluginIndexes.interfaces import IFilteredSet
 from Products.PluginIndexes.interfaces import IPathIndex
 from Products.PluginIndexes.interfaces import IPluggableIndex
 from Products.PluginIndexes.interfaces import ITopicIndex
+from zope.component import adapts
+from zope.component import queryMultiAdapter
```
Comment (dataflake) [inline src/Products/GenericSetup/PluginIndexes/exportimport.py:23]: This is broken. That change should not have been made. Looks like the isort policy doesn't work correctly.

File: src/Products/GenericSetup/ZCTextIndex/exportimport.py [hunk 2]
```diff
@@ -16,11 +16,10 @@
 from BTrees.IOBTree import IOBTree
 from BTrees.Length import Length
 from BTrees.OIBTree import OIBTree
-from zope.component import adapts
-
 from Products.ZCTextIndex.interfaces import IZCLexicon
 from Products.ZCTextIndex.interfaces import IZCTextIndex
 from Products.ZCTextIndex.PipelineFactory import element_factory
+from zope.component import adapts
```
Comment (dataflake) [inline src/Products/GenericSetup/ZCTextIndex/exportimport.py:22]: This is broken. That change should not have been made. Looks like the isort policy doesn't work correctly.

File: src/Products/GenericSetup/ZCatalog/exportimport.py [hunk 3]
```diff
@@ -13,11 +13,10 @@
 """ZCatalog export / import support.
 """
 
+from Products.ZCatalog.interfaces import IZCatalog
 from zope.component import adapts
 from zope.component import queryMultiAdapter
 
-from Products.ZCatalog.interfaces import IZCatalog
-
```
Comment (dataflake) [inline src/Products/GenericSetup/ZCatalog/exportimport.py:20]: This is broken. That change should not have been made. Looks like the isort policy doesn't work correctly.

**Review by dataflake — CHANGES_REQUESTED** [review]
There's a few occurrences where isort sorted incorrectly. Should be a small fix to the isort policy.

File: src/Products/GenericSetup/PluginIndexes/exportimport.py (hunk 1 above)
Comment (icemac) [reply src/Products/GenericSetup/PluginIndexes/exportimport.py]: I forgot to run `isort-apply`, fixed it now.
Maybe we should enforce that the imports are actually sorted by removing the `-` in front of the isort call in `tox.ini`.

Comment (dataflake) [reply src/Products/GenericSetup/PluginIndexes/exportimport.py]: > Maybe we should enforce that the imports are actually sorted by removing the `-` in front of the isort call in `tox.ini`.

Sure, I wouldn't mind that.

**Review by dataflake — APPROVED** [review]

**Pull request merged**